src/functions/OverloadedOoops.kts

```diff
@@ -0,0 +1,29 @@
+package functions.overloadedOoops
+
+class Test {
```

Comment:
this class is unnecessary, top level functions will repro the same; it'll make the puzzle shorter and simpler to understand

Reply:
@TWiStErRob , yes, you are right. I will remove redundant class declaration